src/installer/installer.go

```diff
@@ -26,14 +26,32 @@ import (
 )
 
 const (
-	InstallDir                  = "/opt/install-dir"
-	KubeconfigPathLoopBack      = "/opt/openshift/auth/kubeconfig-loopback"
-	KubeconfigPath              = "/opt/openshift/auth/kubeconfig"
-	minMasterNodes              = 2
-	dockerConfigFile            = "/root/.docker/config.json"
-	assistedControllerPrefix    = "assisted-installer-controller"
-	assistedControllerNamespace = "assisted-installer"
-	extractRetryCount           = 3
+	InstallDir                   = "/opt/install-dir"
+	KubeconfigPathLoopBack       = "/opt/openshift/auth/kubeconfig-loopback"
+	KubeconfigPath               = "/opt/openshift/auth/kubeconfig"
+	minMasterNodes               = 2
+	dockerConfigFile             = "/root/.docker/config.json"
+	assistedControllerPrefix     = "assisted-installer-controller"
+	assistedControllerNamespace  = "assisted-installer"
+	extractRetryCount            = 3
+	assistedInstallerSshManifest = "/opt/openshift/openshift/99_openshift-machineconfig_99-assisted-installer-master-ssh.yaml"
+	sshManifestTmpl              = `
```

Comment:
can we have it as template file maybe? or at least move to another file.

Reply:
This change will later move to `assisted-service`, so no reason to have an extra file for it which will need to be added to the Dockerfile which will lead to downstream-image-change.

Also according to @carbonin, it is short enough to be in code.

This is no different than what we are doing in `assisted-service` which BTW is a much longer file :)
https://github.com/ybettan/assisted-service/blob/be82d501cee20799a4e0d4d95527878886ca6a92/internal/bminventory/inventory.go#L134
https://github.com/ybettan/assisted-service/blob/be82d501cee20799a4e0d4d95527878886ca6a92/internal/bminventory/inventory.go#L199

Reply:
but template sounds good for me

Reply:
so just create new go file with it.

Reply:
this file is already become long, so why to have all this context here?